lib/bolt/transport/ssh/connection.rb

```diff
@@ -232,8 +233,7 @@ def execute(command, sudoable: false, **options)
               sudo_str = Shellwords.shelljoin(sudo_flags)
               command_str = "#{sudo_str} #{command_str}"
             else
-              run_as_str = Shellwords.shelljoin(@target.options['run-as-command'] + [run_as])
-              command_str = "#{run_as_str} #{command_str}"
+              command_str = "#{run_as_cmd} #{command_str}"
```

Comment:
This is a breaking change because it prevents using run-as-command with 'run-as'

Reply:
Yes, but users will just need to update their configs to add the user to run-as-command. It'll be a pain to communicate but we can add an error message about it and I think that's doable, and will allow blocked users to use bolt

Reply:
You can currently pass `_run_as` to individual tasks inside a plan, and can have a plan which 
runs as several users, which this change wouldn't allow.

Reply:
Does that us 'run-as-command'?